src/System.Runtime.Serialization.Formatters/tests/TargetFrameworkMoniker.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    // The values represent platforms where there was change in the serialization for one or more types.
+    public enum TargetFrameworkMoniker
+    {
+        netfx461,
+        netfx471,
+        netfx472,
+        netcoreapp20,
+        netcoreapp21,
+    }
+
+    public static class PlatformExtensions
```

Comment:
Can these be internal rather than public?

Reply:
sure.